docs/usage/index.md

```diff
@@ -76,7 +76,7 @@ Renovate works on these platforms:
 
 Renovate is used by:
 
-![Renovate Matrix](https://app.renovatebot.com/images/matrix.png){ loading=lazy }
+![Renovate Matrix](../usage/assets/images/matrix.png){ loading=lazy }
```

Comment:
As usual, please double check me on the links! :melting_face:

Reply:
```suggestion
![Renovate Matrix](./assets/images/matrix.png){ loading=lazy }
```